### PR TITLE
Creating a PR for TC004

### PR DESCRIPTION
**Pull request by reshmanrahman1**
# What is this PR for?
### Adding New Test Case

# FIle Names Changed in this PR
- Flipkart/src/main/java/com/ecommerce/flipkart/pages/ProductDetailPage.java
- Flipkart/testng.xml

# FIle Names Add in this PR
- Flipkart/src/test/java/com/ecommerce/flipkart/testscripts/TC004.java 

# Jenkins Result
- [joburl](https://www.google.com)

### CODE REVIEW

**Comment by anukilaru** [comment]
looking good

File: Flipkart/src/main/java/com/ecommerce/flipkart/pages/ProductDetailPage.java [hunk 1]
```diff
@@ -86,8 +86,10 @@ public BasePage addItemToCart(boolean proceedToCheckOut)
 		}
 		else
 		{
+			webActionUtil.elementClick(addToCartBtn);
+			Utilities.sleepInSeconds(3);
 			webActionUtil.elementClick(continueToShoppingBtn);
-			return this;
```
Comment (thehemanth) [inline Flipkart/src/main/java/com/ecommerce/flipkart/pages/ProductDetailPage.java:90]: Did you verify whether these changes are not impacting other tests?

**Comment by TamilarasiPrakash** [comment]
No Hemanth

File: Flipkart/src/main/java/com/ecommerce/flipkart/pages/ProductDetailPage.java (hunk 1 above)
Comment (reshmanrahman1) [reply Flipkart/src/main/java/com/ecommerce/flipkart/pages/ProductDetailPage.java:90]: Yes!!
I have verifed the other test in Jenkins after this changes please find the jenkins URL
[Job Result](https://www.google.com)

**Pull request merged**